Babylon/groups/azure/groups/acr/__init__.py

```diff
@@ -0,0 +1,17 @@
+from click import group
+from click import pass_context
+from click.core import Context
+from .commands import list_commands
+from .groups import list_groups
+
+
+@group()
+@pass_context
+def acr(ctx: Context):
+    """Nothing needed here"""
```

Comment:
Need to replace that line by a useful doc-string since it is used by `click` to generate commands help.

As of now it make this result :
```text
└▶ babylon azure
Usage: babylon azure [OPTIONS] COMMAND [ARGS]...

  Group allowing communication with Microsoft Azure Cloud

  Requires key `api_scope` in the platform config file.

Options:
  -h, --help  Show this message and exit.

Commands:
  acr  Nothing needed here
  adt  Allow communication with Azure Digital Twin
  adx  Group interacting with Azure Data Explorer
```